### PR TITLE
develop/add-cmd-table

### PR DESCRIPTION
**Pull request by WinnieXU1**
add the relationship table of InfraSIM cmd and standard server cmd in "Getting started" page

### CODE REVIEW

File: docs/get_start.rst [hunk 1]
```diff
@@ -169,3 +169,30 @@ BOX configuration can be refered on `Packer BOX Configuration <https://github.co
...
 
+Relationship table of Infrasim command and standard server command
+--------------------------------------------------------------------
+
+Here we list a table to reflect the relationship of InfraSIM command and standard server command.
+
+    +---------------------------------------------+-----------------------------+
+    |standard server command                      |InfraSIM command             |
+    +=============================================+=============================+
+    |power on                                     |infrasim node start          |
```
Comment (fub2) [inline docs/get_start.rst]: AC power on and AC power off sounds more reasonable

And would it be more clear if specifying command "sudo infrasim node stop a-node" for purpose of "AC power off a-node"

File: docs/get_start.rst [hunk 2]
```diff
@@ -169,3 +169,30 @@ BOX configuration can be refered on `Packer BOX Configuration <https://github.co
...
+    +---------------------------------------------+-----------------------------+
+    |power off                                    |infrasim node stop           |
+    +---------------------------------------------+-----------------------------+
+    |removal server                               |infrasim node destroy        |
```
Comment (fub2) [inline docs/get_start.rst]: how about "dismiss server" instead of "removal server? Not 100% sure.

File: docs/get_start.rst [hunk 3]
```diff
@@ -169,3 +169,30 @@ BOX configuration can be refered on `Packer BOX Configuration <https://github.co
...
+    +---------------------------------------------+-----------------------------+
+    |reset                                        |infrasim node restart        |
+    +---------------------------------------------+-----------------------------+
+    |KVM - virtual keyboard, visual monitor       |infrasim VNC                 |
```
Comment (fub2) [inline docs/get_start.rst]: "infrasim VNC" is confusing. It is not listing a CLI command like other row does. how about "Connecting to InfraSIM with VNC client"

File: docs/get_start.rst [hunk 4]
```diff
@@ -169,3 +169,30 @@ BOX configuration can be refered on `Packer BOX Configuration <https://github.co
...
+    +---------------------------------------------+-----------------------------+
+    |configuration update                         |1. update yaml               |
```
Comment (fub2) [inline docs/get_start.rst]: "configuration update for a-node"
1. update /a/b/c/a-node.yml"
2.  ...
3. ...
4. ...

File: docs/get_start.rst [hunk 5]
```diff
@@ -169,3 +169,30 @@ BOX configuration can be refered on `Packer BOX Configuration <https://github.co
...
+    |(node type, nic, processor, drive, memory)   |2. infrasim node stop        |
+    |                                             |3. infrasim node destroy     |
+    |                                             |4. infrasim node start       |
+    +---------------------------------------------+-----------------------------+
+    |add new server node                          |1. prepare yml               |
```
Comment (fub2) [inline docs/get_start.rst]: "add new server node b-node"
1. compose b-node.yml
2. infrasim node config add --file=/path/to/b-node.yml
3. start
4. infrasim node config list
both a-node and b-node are there

**Comment by fub2** [comment]
IMHO. @all to take a look and comment if these description is clear.

**Comment by fub2** [comment]
|check node status|infrasim node status a-node|
with explains like this:
If you see a-node-bmc is running, it's AC on
If you see a-node-node is running, it indicates the compute node is power on
@fub2 @xiar please correct the words to see if it's appropriate.

File: docs/get_start.rst [hunk 6]
```diff
@@ -169,3 +169,32 @@ BOX configuration can be refered on `Packer BOX Configuration <https://github.co
         Network Adapter: Bridged Adapter
         Promiscuous Mode: on
 
+Relationship table of Infrasim command and standard server command
+--------------------------------------------------------------------
+
+Here we list a table to reflect the relationship of InfraSIM command and standard server command. Note that the InfraSIM command with (*) here is not the CLI command. Use "infrasim -h" can get the help message.
+
+    +-----------------------------------------------+---------------------------------------------------------+
+    |standard server command                        |InfraSIM command                                         |
```
Comment (fub2) [inline docs/get_start.rst]: Operations on physical server       <-->     corresponding InfraSIM command

File: docs/get_start.rst [hunk 7]
```diff
@@ -169,3 +169,32 @@ BOX configuration can be refered on `Packer BOX Configuration <https://github.co
...
+    |(node type, nic, processor, drive, memory)     |2. infrasim config update a-node [a-node yaml file path] |
+    |                                               |3. infrasim node stop a-node                             |
+    |                                               |4. infrasim node destroy a-node                          |
+    |                                               |5. infrasim node start a-node                            |
+    +-----------------------------------------------+---------------------------------------------------------+
+    |add new server node b-node                     |1. compose b-node yaml file(*)                           |
+    |                                               |2. infrasim config add b-node [b-node yaml file path]    |
+    |                                               |3. infrasim node start b-node                            |
+    |                                               |4. infrasim config list                                  |
+    +-----------------------------------------------+---------------------------------------------------------+
```
Comment (fub2) [inline docs/get_start.rst]: More rows on:

Check server a-node specification                <--->       infrasim node info a-node
Check server a-node running status                          <--->       infrasim node status a-node
                                                                                                  "a-node-bmc is running" AC is on, bmc's alive
                                                                                                   "a-node-node is running" - it indicates the compute                             node is powered on

**Comment by fub2** [comment]
@WinnieXU1 my apology for not adding all comments at one time.

**Comment by turtle-fly** [comment]
Clear enough!

**Pull request merged**